internal/clients/telegram/conversation_state.go

```diff
@@ -29,35 +31,106 @@ type ConversationStateHandler interface {
 // The default state
 type rootConversationState struct{}
 
+func (*rootConversationState) String() string {
+	return "rootState"
+}
+
 func (s *rootConversationState) telegramMessage(message message) (ConversationStateHandler, []telegramBotActor) {
-	log.Printf("Got a message %q", *message.Text)
+	if message.Text != nil {
```

Comment:
looks like debugging code, doesn't it?

Reply:
https://github.com/m-kuzmin/daily-reporter/pull/4/commits/c2f4e1450362c8ab31c1ad1b167e0b28f2edf5f1

Removed the message text, but added prints for what actions the user took.